ipatests/test_integration/test_dns_locations.py

```diff
@@ -557,6 +561,10 @@ def test_adtrust_system_records(self):
             (self.PRIO_HIGH, self.WEIGHT, DNSName(self.master.hostname)),
         )
 
+        for host in (self.replicas[0], self.replicas[1]):
```

Comment:
Same comment as above, the replication goes from master to replicas and the code should read:
```
        ldap = self.master.connect()
        tasks.wait_for_replication(ldap)
```

Reply:
updated to follow the suggestion as well.